docs/source/standard/contract.rst

```diff
@@ -29,90 +29,109 @@ Schema
++-------------------------+---------------------------------------------------------------------------------+
+|        Status           |                             Description                                         |
++=========================+=================================================================================+
+| :`active.confirmation`: | draft contract                                                                  |
++-------------------------+---------------------------------------------------------------------------------+
+| :`active.payment`:      | payment period                                                                  |
++-------------------------+---------------------------------------------------------------------------------+
+| :`active.approval`:     | the period for downloading the final Order on the privatization of the facility |
++-------------------------+---------------------------------------------------------------------------------+
+| :`active`:              | this contract has been signed by all the parties, and is now legally in force   |
++-------------------------+---------------------------------------------------------------------------------+
+| :`active`:              | period for fulfillment of other conditions                                      |
++-------------------------+---------------------------------------------------------------------------------+
+| :`pending.terminated`:  | a vaiting transition to the next status                                         |
++-------------------------+---------------------------------------------------------------------------------+
+| :`pending.unsuccessful`:| a vaiting transition to the next status                                         |
++-------------------------+---------------------------------------------------------------------------------+
+| :`terminated`:          | this contract was signed and in force, and has now come to a close. This may be |
+|                         | due to a successful completion of the contract or may be early termination      |
+|                         | due to some non-completion issue                                                |
++-------------------------+---------------------------------------------------------------------------------+
+| :`unsuccessful`:        | this contract is unsuccessful                                                   |
++-------------------------+---------------------------------------------------------------------------------+
 
 :items:
-    List of :ref:`Item` objects, auto-generated, read-only
+    Array of :ref:`Item` objects, auto-generated, read-only
 
     |ocdsDescription|
-    The goods, services, and any intangible outcomes in this contract. Note: If the items are the same as the award do not repeat.
+    The goods, services, and any intangible outcomes in this contract. Note: If the items are the same as the award do not repeat - to be removed.
```

Comment:
Note: If the items are the same as the award do not repeat - to be removed. 
this sentence has to be **DELETED**